src/main/java/edu/cornell/kfs/module/ld/batch/service/impl/CuFileEnterpriseFeederHelperServiceImpl.java

```diff
@@ -106,7 +107,7 @@ public void feedOnFile(File doneFile, File dataFile, File reconFile, PrintStream
                     
                 Collection<String> offsetDocTypes = parameterService.getParameterValuesAsString(
                         LaborEnterpriseFeedStep.class, LaborParameterConstants.LABOR_BENEFIT_OFFSET_DOCTYPE);
-                offsetDocTypes = offsetDocTypes.stream().map(String::toUpperCase).collect(Collectors.toList());
+                offsetDocTypes = offsetDocTypes.parallelStream().map(offsetDocType -> offsetDocType.toUpperCase(Locale.US)).collect(Collectors.toList());
```

Comment:
Did you intentially switch from stream to parallelStream?

Reply:
Thanks for catching.